manager/assets/modext/widgets/security/modx.grid.access.policy.js

```diff
@@ -71,7 +71,10 @@ MODx.grid.AccessPolicy = function(config) {
             header: _('description')
             ,dataIndex: 'description'
             ,width: 375
-            ,editor: { xtype: 'textarea' }
+            ,renderer: function(value, metaData, record) {
+                return record['data']['description_trans'];
```

Comment:
By defining a custom renderer, the standard renderer to encode html is no longer applied. Do we consider the translation lexicon safe or user input?

Reply:
Added encoding to avoid potential XSS.